src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -139,8 +139,7 @@ private boolean applyExistingArtefact(Artefact artefact) {
             artefact.setArtefactId(value.getArtefactId());
             artefact.setPayload(value.getPayload());
             artefact.setSupersededCount(value.getSupersededCount() + 1);
-            if (!artefactService.payloadWithinLimit(artefact.getPayloadSize())
-                && artefactService.payloadWithinLimit(value.getPayloadSize())) {
+            if (!artefactService.payloadWithinLimit(artefact.getPayloadSize())) {
```

Comment:
After considering a few options for this to prevent the scenario where a publication already exists that is > the new limit (adding another temporary flag with the 'old' limit for a period of time until all old publications were deleted or checking channel management using the filesExists endpoint to see if there are files before deleting) I ended up going with the removal of this for a few reasons:

1. Adding another variable / temporary flag for the 'old' value seemed prone to error and added extra complexity - if the old flag was set lower than the new flag it would mean payloads might not get deleted

2. Calling Channel Management to check if the files exist first was nearly the solution I went with, however it felt like a slightly unnecessary due to 1) the delete already handling if the file doesn't exist gracefully and 2) it would require a new call to channel management which would then soon be changed with the merge